src/pages/SettingsPage/PreferencesPage.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {View} from 'react-native';
+import RNPickerSelect from 'react-native-picker-select';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import {redirect} from '../../libs/actions/App';
+import HeaderGap from '../../components/HeaderGap';
+import ROUTES from '../../ROUTES';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import Icon from '../../components/Icon';
+import NameValuePair from '../../libs/actions/NameValuePair';
+import CONST from '../../CONST';
+import {DownArrow} from '../../components/Icon/Expensicons';
+
+
+const propTypes = {
+    // The chat priority mode
+    priorityMode: PropTypes.string,
+};
+
+const defaultProps = {
+    priorityMode: CONST.PRIORITY_MODE.DEFAULT,
+};
+
+const priorityModes = {
+    default: {
+        value: CONST.PRIORITY_MODE.DEFAULT,
+        label: 'Most Recent',
+        description: 'This will display all chats by default, sorted by most recent, with pinned items at the top',
+    },
+    gsd: {
+        value: CONST.PRIORITY_MODE.GSD,
+        label: 'GSD',
+        description: 'This will only display unread and pinned chats, all sorted alphabetically. Get Shit Done.',
+    },
+};
+
+const PreferencesPage = ({priorityMode}) => (
+    <>
+        <HeaderGap />
+        <HeaderWithCloseButton
+            title="Preferences"
+            onCloseButtonPress={() => redirect(ROUTES.SETTINGS)}
+        />
+        <View style={styles.settingsWrapper}>
+            <View style={[styles.settingsPageBody, styles.mt4, styles.mb6]}>
```

Comment:
Since this is the very first thing in the view, I think we can get rid of the `mt4` as we don't need any margin top here.